### PR TITLE
Improve post cell header

### PR DESCRIPTION
**Pull request by vadymmarkov**

### CODE REVIEW

File: Source/Nodes/AttachmentGridNode.swift [hunk 1]
```diff
@@ -9,16 +9,20 @@ public class AttachmentGridNode: ASControlNode {
   var imageNodes = [ASImageNode]()
   var counterNode: CounterNode?
 
+  private var config: Config.Wall.Post.Attachments.Type {
+    return Config.Wall.Post.Attachments.self
```
Comment (zenangst) [inline Source/Nodes/AttachmentGridNode.swift]: Nifty! Short hand-method for less code, I like it.

**Comment by zenangst** [comment]
@vadymmarkov I think we could start adding some pragma marks? What do you think?

**Comment by vadymmarkov** [comment]
@zenangst Agreed. Will start from the next PR and also will do some refactoring for all the code in another PR.

**Comment by zenangst** [comment]
@vadymmarkov made a issue for it ;) https://github.com/hyperoslo/Wall/issues/33

**Pull request merged**